### PR TITLE
Add buttons to control audio playback

### PR DESCRIPTION
**Pull request by LittleHuba**

### CODE REVIEW

**Comment by LittleHuba** [comment]
@andreasb242 would you mind taking a look at the way I'm updating the ui elements with the approriate state?

**Comment by andreasb242** [comment]
I'm already reviewing...

File: src/gui/toolbarMenubar/ToolMenuHandler.cpp [hunk 1]
```diff
@@ -478,7 +478,11 @@ void ToolMenuHandler::initToolItems()
 	fontButton = new FontButton(listener, gui, "SELECT_FONT", ACTION_FONT_BUTTON_CHANGED, _("Select Font"));
 	addToolItem(fontButton);
 
-	ADD_CUSTOM_ITEM_TGL("RECSTOP", ACTION_RECSTOP, GROUP_REC, false, "rec", _("Rec / Stop"));
+	ADD_CUSTOM_ITEM_TGL("AUDIO_RECORDING", ACTION_AUDIO_RECORD, GROUP_AUDIO, false, "audio-record", _("Record Audio / Stop Recording"));
```
Comment (andreasb242) [inline src/gui/toolbarMenubar/ToolMenuHandler.cpp]: If you change something like this, we need backward compatibility.... (not checked yet)

File: src/util/audio/AudioPlayer.cpp [hunk 2]
```diff
@@ -25,44 +26,78 @@ AudioPlayer::~AudioPlayer()
...
-								 while (this->portAudioConsumer->isPlaying())
-								 {
-									 Pa_Sleep(100);
-								 }
-								 this->stop();
-							 });
-	stopThread.detach();
+	if (status)
+	{
+		status &= this->play();
```
Comment (andreasb242) [inline src/util/audio/AudioPlayer.cpp]: This probably should be logical or? Even if this probably works in 99% of all the cases with bool....

File: src/gui/toolbarMenubar/ToolMenuHandler.cpp (hunk 1 above)
Comment (LittleHuba) [reply src/gui/toolbarMenubar/ToolMenuHandler.cpp]: Where and how do I add this?

Comment (andreasb242) [reply src/gui/toolbarMenubar/ToolMenuHandler.cpp]: ToolMenuHandler::load, there is already such a backward fix:

```
				// recognize previous name, V1.07 (Jan 2019) and earlier.
				if (name == "TWO_PAGES")
				{
					name = "PAIRED_PAGES";
				}
```

**Comment by andreasb242** [comment]
Locks good!

**Pull request merged**